common/changes/@uifabric/styling/edwl-2018-10-fixCoachmarkDelay_2018-10-03-21-59.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "add delayLength parameter to _createDefaultAnimation in PulsingBeaconAnimationStyles",
+      "type": "patch"
```

Comment:
@dzearing is this considered a `patch` or `minor` change?

Reply:
Typically any `*.types.ts` change is a `minor` bump unless its a breaking API change.